src/components/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { Status } from '../types/Status';
+
+type Props = {
+  todos: Todo[],
+  selectItem: Status,
+  setSelectItem: React.Dispatch<React.SetStateAction<Status>>
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  selectItem,
+  setSelectItem,
+}) => {
+  const isActiveTodos = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${isActiveTodos.length} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: selectItem === Status.ALL,
+          })}
+          onClick={() => setSelectItem(Status.ALL)}
```

Comment:
consider to make one handler factory for all filter buttons and call in jss with proper field
```
const handleSelect = (status: Status) => () => setSelectItem(status);
```
```suggestion
          onClick={handleSelect(Status.ALL)}
```